src/store/apis/hotels/getPopularHotels.endpoint.js

```diff
@@ -0,0 +1,11 @@
+import { hotelsApi } from './hotels.api';
+
+export const homesApi = hotelsApi.injectEndpoints({
```

Comment:
I'd like to suggest you to rename it and remove the export
```suggestion
const getPopularApi = hotelsApi.injectEndpoints({
```